addons/wowl/static/src/components/user_menu/user_menu.ts

```diff
@@ -27,12 +27,16 @@ export class UserMenu extends Component<{}, OdooEnv> {
   static components = { Dropdown, DropdownItem };
   source: string;
   user = useService("user");
+  displayName = this.user.name;
 
   constructor() {
     super(...arguments);
     const { origin } = this.env.browser.location;
     const { userId } = this.user;
     this.source = `${origin}/web/image?model=res.users&field=image_128&id=${userId}`;
+    if (this.env.debug) {
```

Comment:
I would have put this in the template
`<t t-if="env.debug" t-esc="dbName" />`

Reply:
Like in the last commit ?